### PR TITLE
Reduce yaml files usage

### PR DESCRIPTION
**Pull request by JRivrain**
Reduce the amount of yaml schedules for select_modules_and_patterns+registration.

See https://progress.opensuse.org/issues/153069
and 
https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/37

VRs https://openqa.suse.de/tests/overview?distri=sle&version=15-SP6&build=JRivrain%2Fos-autoinst-distri-opensuse%2318448

### CODE REVIEW

File: schedule/yast/select_modules_and_patterns+registration/select_modules_and_patterns+registration.yaml [hunk 1]
```diff
@@ -10,44 +10,31 @@ description: >
         report any issues;
```
Comment (JRivrain) [inline schedule/yast/select_modules_and_patterns+registration/select_modules_and_patterns+registration.yaml:10]: Looks like a modification of select_modules_and_patterns+registration.yaml, but it's a merge of select_modules_and_patterns+registration_x86_64.yaml and select_modules_and_patterns+registration_aarch64.yaml

**Comment by jknphy** [comment]
we can also move the YUI parameter, we can find a way in the CI to reuse and perhaps later to remove it definitevely.
the resulting yaml will be here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/schedule/yam/test_cases please move it there, should be in some of the tickets of the epic (I hope) if now we should update or add something here:
https://confluence.suse.com/display/qasle/How-To%3A+Reduce+number+of+the+yaml+schedule+files

**Comment by JRivrain** [comment]
> we can also move the YUI parameter, we can find a way in the CI to reuse and perhaps later to remove it definitevely. the resulting yaml will be here: https://github.com/os-autoinst/os-autoinst-distri-opensuse/tree/master/schedule/yam/test_cases please move it there, should be in some of the tickets of the epic (I hope) if not we should update or add something here: https://confluence.suse.com/display/qasle/How-To%3A+Reduce+number+of+the+yaml+schedule+files

I don't know how to use CI to setup parameters. I could put YUI parameter in the jobgroup's yaml, but then I need to repeat it twice, once for aarch64 and once for x86_64. Do you mean that ?

**Comment by jknphy** [comment]
the goal is to have one yaml test suite under /test_cases/ if we cannot make it for some reason we shouldn't move the file to that location, so we can see in the future where we made it and where we didn't (and we might need some more coding somewhere to achieve it). Please, move them back until you have a solution with single yaml.

**Comment by JRivrain** [comment]
> the goal is to have one yaml test suite under /test_cases/ if we cannot make it for some reason we shouldn't move the file to that location, so we can see in the future where we made it and where we didn't (and we might need some more coding somewhere to achieve it). Please, move them back until you have a solution with single yaml.
> 
> We can unify what we can here (not moving files) and then produce an analysis of the modules that are giving us a headache and put that analysis in a follow-up ticket, or deal with each difference in this one, up to you, it is a a bit of a sw archaeological task at this point, but still worth it.

There are several challenges here, for example, we need svirt_upload_assets only for s390-kvm, we are validating in yast for aarch64 and x86 but not fo the other backends, and more... I think we cannot always have only one file for all at the moment. I will see what I can do but this is quite a mess.

**Pull request merged**